evil-man.el

```diff
@@ -48,7 +48,7 @@
 
     ;; goto
     "gm" 'man
-    "gd" 'Man-goto-section
+    "gd" 'Man-goto-section ; TODO: This is not a "go to definition".
```

Comment:
This seems like a stray TODO?

Reply:
Bad phrasing, but I meant here that we should change the binding: this has nothing to do with "go to definition".

Should we define a standard binding for commands "listing stuff/sections"?